indigo_app/templates/indigo_api/work_new_batch.html

```diff
@@ -104,14 +129,13 @@ <h2 class="mt-3 mb-3">Create a new batch of works</h2>
           <a href="{% url 'workflow_create' place=place.place_code %}" class="btn btn-primary" target="_blank">Create workflow</a>
         </div>
 
-        {% if place_workflows %}
+        {% if form.fields.workflow.choices %}
           <div class="form-group">
             <div class="row">
               <div class="col-md-6">
-                <select class="selectpicker form-control" required name="{{ form.workflow.name }}" id="{{ form.workflow.id_for_label }}" data-live-search="true">
-                  <option>Choose one</option>
-                  {% for workflow in place_workflows %}
-                    <option value="{{ workflow.id }}" {% if workflow.id in form.workflow.value %}selected{% endif %}>{{ workflow.title }}</option>
+                <select class="selectpicker form-control" required name="{{ form.workflow.name }}" id="{{ form.workflow.id_for_label }}" data-live-search="true" placeholder="Choose a workflow">
```

Comment:
`placeholder` doesn't work

Reply:
Oh it's selectpicker. Try `data-liveSearchPlaceholder="foo"`?

Reply:
this gives a placeholder in the search box, but the first workflow is still selected by default, which we don't want
![image](https://user-images.githubusercontent.com/32566441/61862150-afa70c80-aecd-11e9-8b40-56c47c23d373.png)

Reply:
Ah, use `title`:

https://developer.snapappointments.com/bootstrap-select/examples/ "Using the title attribute will set the default placeholder text when nothing is selected. This works for both multiple and standard select boxes:
"

Reply:
I suspect we may have to do something to the form for this to take; just replacing `placeholder` with `title` didn't do anything – the first workflow is still selected by default :/